js/ui/drop_down_box.js

```diff
@@ -249,11 +249,7 @@ var DropDownBox = DropDownEditor.inherit({
         this.callBase();
 
         if(this.option("focusStateEnabled")) {
-            this._popup._keyboardProcessor.push(new KeyboardProcessor({
-                element: this.content(),
-                handler: this._popupElementTabHandler,
-                context: this
-            }));
+            keyboard.on(this.content(), null, e => this._popupElementTabHandler(e));
```

Comment:
I found that any other keyboard subscriptions stores an IDs to detach them later. Why it does not happened here? Maybe it discovers a leak? If so, maybe it's possible to to put these id's on each subscription automatically? It could allow to detach them all on disposing.